libs/antsi/src/lib.rs

```diff
@@ -1,14 +1,500 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
+// Good reference to begin with: https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797
+#![no_std]
+#![cfg_attr(
+    nightly,
+    feature(provide_any, error_in_core, error_generic_member_access)
```

Comment:
For what do we need these?

Reply:
we don't, will remove.

Reply:
fixed in [4cf4e0a](https://github.com/hashintel/hash/pull/1995/commits/4cf4e0a89d1d72308e8cbd094731bbf04c0d46dc)